README.md

```diff
@@ -53,9 +53,9 @@ git-subtree. See those directories' README files for more details.
 
 ## See Also
 
-* [XRP Ledger Dev Portal](https://developers.ripple.com/)
-* [XRP News](https://ripple.com/category/xrp/)
-* [Setup and Installation](https://developers.ripple.com/install-rippled.html)
+* [XRP Ledger Dev Portal](https://xrpl.org/)
+* [XRP News](https://blog.ripplex.io/xrp/)
```

Comment:
I know this was mostly a mechanical rename, but this URL is dead and also, it's just RippleX's blog. I don't know if it needs to be in here, but even if it is, it should not be named "XRP News". Honestly, I'd rather just remove it.

Reply:
+1 for removing it